docs/static/ds6/skin.css

```diff
@@ -359,12 +359,15 @@ svg.icon--close {
   width: 17px;
 }
 span.icon {
+  background-repeat: no-repeat;
   display: inline-block;
   vertical-align: middle;
-  background-repeat: no-repeat;
 }
 span.icon--close {
-  background-image: url('data:image/svg+xml,%3Csvg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%2017%2017%22%3E%3Cpath%20fill=%22none%22%20stroke=%22currentColor%22%20stroke-linecap=%22round%22%20stroke-linejoin=%22round%22%20stroke-width=%222%22%20d=%22M1.11%201.78l14.15%2014.15m-14.09-.07L15.32%201.72%22/%3E%3C/svg%3E');
+  background-repeat: no-repeat;
+  display: inline-block;
+  vertical-align: middle;
+  background-image: url('data:image/svg+xml,%3Csvg%20xmlns%3D%27http%3A//www.w3.org/2000/svg%27%20width%3D%2717%27%20height%3D%2717%27%3E%3Cpath%20fill=%22none%22%20stroke=%22currentColor%22%20stroke-linecap=%22round%22%20stroke-linejoin=%22round%22%20stroke-width=%222%22%20d=%22M1.11%201.78l14.15%2014.15m-14.09-.07L15.32%201.72%22%3E%3C/path%3E%3C/svg%3E');
```

Comment:
I have a question, do we follow a certain order for styling? e.g. Alphabetical

Reply:
Yes, alpha should be the way to go but the LESS files may end up compiling to _not_-alpha.